setup.py

```diff
@@ -115,9 +115,8 @@ class Sdist(sdist):
         def __init__(self, *args, **kwargs):
             raise Exception('Cython is required to run sdist')
 
-    if not on_rtd and not all(os.path.exists(path) for path in [
-        compress_source, crypto_ll_source, chunker_source, hashindex_source, item_source, checksums_source,
-        platform_posix_source, platform_linux_source, platform_freebsd_source, platform_darwin_source]):
+    cython_c_files = map(lambda fn: fn.replace('.pyx', '.c'), cython_sources)
```

Comment:
Guess a list comprehension is easier here than map/lambda:
```
cython_c_files = [fn.replace('.pyx', '.c') for fn in cython_sources]
```

It even could be put into the following line by just adding the `.replace(...)` there.

Reply:
I'll change that in a minute though I have a slight preference for two lines (I think the code is more expressive if readers see easily that this is a about "cython files (names) in C". Also I had in mind that we might have to do this for "cpp" if msgpack needs to be bundled in 1.2.0.

Reply:
yeah, 2 lines is more clear.